### DISCUSSION TITLE
S3 Upload of HLS files uploads 1.30min ts files for Conferencing

### DISCUSSION THREAD

**Discussion by lexy10** [ant-media/Ant-Media-Server#6870] [Q&A]
I have been stuck with configuration settings on s3 upload of hls file when I'm using conferencing, but the ts files on s3 are 90seconds long, no matter the settings i do. 

<img width="677" alt="Screenshot 2024-12-12 at 11 34 21 AM" src="https://github.com/user-attachments/assets/5270b3e5-0600-416e-a643-4f5ae9d7a635" />

**Comment by filip-ams** [comment]
Hi @lexy10,

Could you provide a bit more detail regarding your setup? The specifications of you instance etc?

**Reply by filip-ams** [reply]
I will close this due to inactivity, feel free to reopen. @lexy10
